docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+Data Assets can be further partitioned into Batches.  Batches are unique subsets of records within a Data Asset.  For example, say you have a Data Asset in a SQL Datasource that consisted of all records from last year in a given table.  You could then partition those records into Batches of data that correspond to the records for individual months of that year.
+
+Using Batches to further partition a Data Asset is optional.  However, having a list for Batches in your Data Asset is not.  This is because GX uses Batches to specify the data to work with.  By default, if you do not further partition the data in a Data Asset it will contain a list with only one Batch.  In that case, the single Batch will consist of all the records in that Data Asset.
+
+- [Data Asset and Batch tutorials](/docs/guides/connecting_to_your_data/manage_data_assets_lp)
+
+### Batch Requests
+
+A Batch Request specifies one or more Batches within the Data Asset.  Batch Requests are the primary way of retrieving data for use in GX.  Because Batch Requests can specify multiple Batches they provide significant flexibility in how you utilize the data in a single Data Asset.
+
+As an example, GX can automate the process of running statistical analyzes for multiple Batches of data.  This is possible because you can provide a Batch Request that corresponds to multiple Batches in a Data Asset.  Alternatively, you can specify a single Batch from that same Data Asset so that you do not need to re-run the analyzes on all of your data when you are only interested in a single subset.
+
+Taking the example of a Data Asset that has been partitioned into Batches by months, this would let you build a statistical model off of each month of your existing data by providing all of the Batches in a single Batch Request.  Then, after an update, you could specify that you only wanted to run your analysis on the most recent month's data by providing a Batch Request that only indicates that one Batch.
+
+- [Batch Request tutorial](/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset)
+
+## Expectations
+
+An Expectation is a verifiable assertion about Source Data.  Similar to assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behaviors. Unlike traditional unit tests which describe the expected behavior of code given a specific input, Expectations apply to the input data itself. For example, you can define an Expectation that a column contains no null values. When GX runs that Expectation on your data it generates a report which indicates if a null value was found.
```

Comment:
These two paragraphs read like general value prop for GX. I think keeping the first paragraph is fine or combine these two into one. We should have communicated this earlier in the funnel.

Reply:
Second paragraph removed.